### DISCUSSION TITLE
Embed linked issue description in PR Conversation tab

### DISCUSSION THREAD

**Discussion by 0xA-10** [community/community#36588] [Pull Requests]
For example:

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/70483566/196537606-774a259c-ad97-4919-8a2d-9ae744427f30.png">

This could also be hidden behind a "Show description" spoiler for example. A few different ways this idea could be implemented, but I just want an easy and obvious way to view the linked issue's full description without leaving the conversation view.

---
old description:
~~I have often noticed myself or my team members copy pasting linked issue descriptions into their PR description. While some may argue this is bad practice, personally I see the issue description to be very relevant to the PR Conversation. It would be very convenient if any linked issues' descriptions could be embedded somewhere easily accessible at a glance in the Conversation tab. Currently I have to open the linked issue from the sidebar to access this information, or copy paste it into the PR description.~~

**Comment by haseeb-xd** [comment]
For me, easiest way is to write issue name after `#`, It will show you the list of issues with that specific name you have written. You can see the example in this photo: 
![image](https://user-images.githubusercontent.com/47222685/196534835-77c89b4e-dedd-4250-955a-47465ab3b0fb.png)

**Reply by 0xA-10** [reply]
That is a nifty feature I was unaware of, thank you for sharing. However, I think what I am proposing is a little different. The important bit is to have the linked issue's description embedded somewhere in the PR conversation. For example:

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/70483566/196537606-774a259c-ad97-4919-8a2d-9ae744427f30.png">

This could also be hidden behind a "Show description" spoiler for example. A few different ways this idea could be implemented, but I just want an easy and obvious way to view the linked issue's full description without leaving the conversation view.

edit: adjusting the main post to incorporate this explanation

**Comment by ossanna16** [comment]
Hey there @anatelli10 and thank you for asking a good question! I'm glad you found our community 🙂
